### ISSUE TITLE
Feature/collaboration request - export posts to RSS

### ISSUE THREAD

**Issue by v-ko** [matt-kruse/SocialFixer#15]
TLDR: Would it be possible to use SocialFixer components to extract FB posts as RSS? No matter whether as a SocialFixer feature or a separate extension. I'd like to collaborate on that.

Hello! Social media is pretty cluttered and uncontrollable . At this point it's a public fact that there are great amounts of behavioral research poured into grabbing our attention in the social media sites. That's probably the reason SocialFixer exists.
 AFAIK RSS streams are the most universal form of content consumption and there are already a lot of tools around them. Having the ability to get FB content as RSS and filter it would be a game changer in the field, albeit probably not the most user friendly experience at first.
